haystack/pipelines/ray.py

```diff
@@ -48,18 +47,23 @@ class RayPipeline(Pipeline):
             |              inputs: [ Query ]
             ```
 
-    A RayPipeline can only be created with a YAML Pipeline config.
-    >>> from haystack.pipeline import RayPipeline
-    >>> pipeline = RayPipeline.load_from_yaml(path="my_pipelines.yaml", pipeline_name="my_query_pipeline")
-    >>> pipeline.run(query="What is the capital of Germany?")
+    A Ray Pipeline can only be created with a YAML Pipeline configuration.
 
-    By default, RayPipelines creates an instance of RayServe locally. To connect to an existing Ray instance,
+    ```python
+    from haystack.pipeline import RayPipeline
+    pipeline = RayPipeline.load_from_yaml(path="my_pipelines.yaml", pipeline_name="my_query_pipeline")
+    pipeline.run(query="What is the capital of Germany?")
+    ```
+
+    By default, RayPipelines create an instance of RayServe locally. To connect to an existing Ray instance,
     set the `address` parameter when creating the RayPipeline instance.
+
+    You can validate the YAML syntax of your Ray Pipelines. For more information, see [YAML File Definitions](https://haystack-website-git-fork-fstau-dev-287-search-deepset-overnice.vercel.app/components/pipelines#yaml-file-definitions).
```

Comment:
Rather then `You can validate the YAML syntax of Ray pipelines.` I'd state it as `YAML definitions of Ray pipelines are validated at load.` Actually validation can't be disabled for now, so it's not optional.

Reply:
Thanks! added